dotCMS/src/main/java/com/dotcms/storage/MetadataGeneratorImpl.java

```diff
@@ -64,6 +65,7 @@ public Map<String, Serializable> tikaBasedMetadata(final File binary, final long
     public TreeMap<String, Serializable> standAloneMetadata(final File binary){
         final TreeMap<String, Serializable> metadataMap = new TreeMap<>(Comparator.naturalOrder());
         final String binaryName = binary.getName();
+        final String mimeType = Try.of(()->MimeTypeUtils.getMimeType(binary)).getOrElse(FileAsset.UNKNOWN_MIME_TYPE);
```

Comment:
Should we use `application/octet-stream` for our unknown mimeType?  Seems safer?